### ISSUE TITLE
How to save empty ng-tags-input as array?

### ISSUE THREAD

**Issue by sinall** [vitalets/angular-xeditable#766]
If the model is not predefined, is there a way to save empty ng-tags-input as array?
Here's the [demo](http://jsfiddle.net/9hs7njpx/6/) showing that ng-tags-input leaves model as undefined.

**Comment by ckosloski** [comment]
Yeah, there isn't really a good way to overwrite the model data on save (see comments in #736)

Resetting the model in `save` seems like your best bet.

**Comment by sinall** [comment]
@ckosloski So firstly should xeditable inputs save to model regardless the input text/value is empty or not? I mean do you consider this behavior a defect or it is by design?
Currently I reset the model in save method, I was expecting the ng-tags-input do it implictly.
